css/main.css

```
@@ -1,16 +1,10 @@
 @font-face {
-  font-family: 'Proxima Nova';
-  src: local('Proxima Nova'), url(../fonts/11096.otf) format('opentype');
-}
-
-@font-face {
-  font-family: 'Open Sans';
-  src: local('Open Sans'), url(../fonts/OpenSans-Light.ttf) format('truetype');
-}
-
-@font-face {
-  font-family: 'Lato';
-  src: local('Lato'), url(../fonts/Lato-Regular.ttf) format('truetype');
+  font-family: 'proxima_novabold';
+  src:
+    url('../fonts/proxima_novabold-webfont.woff2') format('woff2'),

```

Let's use local font when possible so downloading can be skipped
```suggestion
    local('Proxima Nova'),
    url('../fonts/proxima_novabold-webfont.woff2') format('woff2'),
```

---

@liubich What do you think about that?